test/plugins/features/movies/index.test.js

```diff
@@ -20,4 +20,73 @@ describe('movies integration', () => {
 
   });
 
+  describe('retrieve movies', () => {
+
+    it('retrieve all movies', () => {
+      return Movies.inject({
+        url: '/movies',
+        method: 'GET',
+        payload: {}
```

Comment:
Do you need this payload here? Since it's a `GET` request I think it might follow the pattern of below where there's no payload specified.

Reply:
Ah yes, thought I removed it in all places. :(